src/createRoom.ts

```diff
@@ -141,6 +144,10 @@ export default async function createRoom(opts: IOpts): Promise<string | null> {
                     [EventType.RoomServerAcl]: 100,
                     [EventType.RoomEncryption]: 100,
                 },
+                users: {
+                    // Temporarily give ourselves the power to set up a widget
+                    [client.getUserId()]: 200,
+                },
```

Comment:
Can we not set it up using initial_state instead of after creation?

Reply:
Ah, that's indeed a better solution

Reply:
Well, on further examination it appears that creating the widget in `initial_state` would prevent video rooms from using the Prosody-Matrix auth plugin, since that depends on deriving a conference ID from the room ID.